### PR TITLE
Recipe app project.

### PR DESCRIPTION
**Pull request by J2ZROMERO**
- Use Ruby gems as a software packages system.
- Install Ruby on Rails framework.
- Use controllers to handle requests and render empty views.
- Understand Rails naming conventions.
- Use params from browser requests in a safe way.
- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.
- Use database migration files to maintain database schema.
- Use validations for models.
- Secure app from n+1 problems.
- Understand what ORM is.
- Write SQL queries with ActiveRecord.
- Set up associations between models.
- Build a web app that requires the user to log in.
- Use Devise gem for authentication.

https://user-images.githubusercontent.com/63516282/219850219-df018876-86cd-4e6c-bccc-56667128c4d1.mp4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,36 +254,21 @@ Feel free to check the [issues page](../../issues/).
...
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+“A recipe has no soul. You as the cook must bring soul to the recipe.”
+– Thomas Keller
```
Comment (rotimiazeez) [inline README.md:266]: Just so you know, the MIT license link provided below is invalid. Kindly fix it 😉

File: spec/requests/foods_spec.rb [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      end.to change(Food, :count).by(-1)
+    end
+
+    it 'redirects to the foods list' do
+      food = Food.create! valid_attributes
+      delete food_url(food)
+      expect(response).to redirect_to(foods_url)
+    end
+  end
+end
```
Comment (rotimiazeez) [inline spec/requests/foods_spec.rb]: Please note that most of your tests haven't been implemented yet which made it appear as pending when the tests are been checked. 👍🏽 Kindly see to this
Screenshot from 2023-02-18 11-17-04

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Great teamwork 🤝 
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
### 
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by J2ZROMERO** [comment]
- Update readme.md file

**Review by m05e5 — APPROVED** [review]
**Hi @J2ZROMERO and @Oklukeok 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Nice UI
✔️ No linters error
✔️ Clean and well-organized code

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**